ui/components/show/showCards/ShowCards.tsx

```diff
@@ -12,72 +11,74 @@ import CardImgContainer from './ImgContainer/CardImgContainer';
 import TagsContainer from '../../layout/cardElements/TagsContainer/TagsContainer';
 import ContentSection from '../../layout/ContentSection/contentSection';
 
-import styles from "./ShowCards.module.scss";
+import { Swiper, SwiperSlide } from 'swiper/react';
+import 'swiper/swiper-bundle.min.css';
```

Comment:
@shawnCaza I have added the imports according to the docs now, please review.

Actually, I am new to this open source stuff and have only some basic experience in React and TS  so, please forgive me if I am making some silly mistakes.

Reply:
@ashish0kumar No problem, I understand you're new.  I don't think you're making any mistakes in regards to React or TS. The whole PR is based on incorporating the 3rd party swiper library. If you haven't worked with Swiper before it would be really difficult to just guess how to do it. You really need to be able to go through the reference material if you're new to Swiper.

I think part of what might be making this difficult is that this particular library is not just for react. The steps for using it would be slightly different if using vanilla JS or Vue. That's why I linked to the React specific docs to help us get on the same page.

You got the right imports from the pagination demo for the core, and pagination styles. We still need the import A11y. It's not in the pagination demo, but follows the same format and is listed in Swiper docs for react link above.